runelite-client/src/main/java/net/runelite/client/plugins/slayer/Task.java

```diff
@@ -136,6 +136,7 @@
 	NECHRYAEL("Nechryael", ItemID.NECHRYAEL, "Nechryarch"),
 	OGRES("Ogres", ItemID.ENSOULED_OGRE_HEAD, "Mogre", "Ogress", "Skogre", "Zogre"),
 	OTHERWORLDLY_BEING("Otherworldly beings", ItemID.GHOSTLY_HOOD),
+	PHANTOM_MUSPAH("Phantom Muspah", ItemID.ANCIENT_ESSENCE),
```

Comment:
We generally use the pet item for bosses

Reply:
Good point, fixed